integration/integration_suite_test.go

```diff
@@ -32,6 +32,7 @@ var (
 	endpoint            = os.Getenv("S3_ENDPOINT")
 	v2signing           = os.Getenv("S3_V2_SIGNING")
 	pathStyle           = len(os.Getenv("S3_USE_PATH_STYLE")) > 0
+	skipS3Checksums     = len(os.Getenv("S3_SKIP_S£_CHECKSUMS")) > 0
```

Comment:
S£?

Reply:
Good catch. I fixed (and feel a bit bad for leaking my keyboard layout and poor typing skills).